remediation/machine-deletion-remediation/known-issues.md

```diff
@@ -0,0 +1,12 @@
+---
+layout: default
+title: Known Issues
+parent: Machine Deletion Remediation
+grand_parent: Remediation
+nav_order: 3
+---
+
+# Known Issues
+Machine Deletion known issues:
+1. NHC creates a redundant Machine Deletion Remediation immediately after a successful remediation [Bug](https://issues.redhat.com/browse/ECOPROJECT-205)
```

Comment:
IIRC we had some discussion on that and this issue does not replicate anymore.
If that is indeed the case I think we can close the issue and remove this line.

Feel free to disregard this comment if you think further investigation is needed.

Reply:
I recall the discussion, but I haven't had time to try replicate the problem, so AFAIK it might be still there